crates/walrus-sui/src/client/retry_client/retriable_rpc_client.rs

```diff
@@ -525,7 +561,7 @@ impl RetriableClientError {
     /// The time window during which failures are counted.
     const FAILURE_WINDOW: Duration = Duration::from_secs(300);
     /// The maximum number of failures allowed.
-    const MAX_FAILURES: usize = 100;
+    const MAX_FAILURES: usize = 20;
```

Comment:
From above, it's unclear why we are moving retry logic into the failover logic.

Reply:
Reverted this change. Changes related to this configure change is now handled in #2090